Source/UI/NewUI/PaymentForm/StyleProtocol/PaymentHeaderCellStyle.swift

```diff
@@ -9,8 +9,8 @@
 import UIKit
 
 public protocol PaymentHeaderCellStyle {
+    var isIconsViewHidden: Bool { get }
```

Comment:
The naming in my opinion is leaking implementation details. Also is being negative whereas I believe we should always strive for positive details. My requirements for this name would be:
- don't mention icons view or UI components
- try to be positive and not negative (show vs hide, enable vs disable)

My suggestion would still be something in the lines of `showAcceptedCardsList`, but happy for other ideas as long as they tick those 2 points